devotionals/extreme-faith/RUNBOOK.md

```
@@ -0,0 +1,130 @@
+5. **Publish** — load credentials and run without `--dry-run`:
+   ```bash
+   set -a && source .env.production && set +a
+   python3 skills/custom-devotional-crafter/scripts/push_custom_devotional.py --file /tmp/maranatha-payload.json
+   ```
+   Upserts on `for_date`. The iOS app renders the tappable "Custom" badge (with attribution alert) plus a context line: *"Extreme Faith · Week N"*.
+
+---
+
+## File layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)